ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-commands.sh

```
@@ -265,6 +265,42 @@ if [[ $IS_ACTIVE_CLUSTER_OPENSHIFT != "false" ]]; then
+echo "To access the cluster:"
+echo "  1. Find this pod in Prow job logs"
+echo "  2. oc rsh -n test-credentials <pod-name>"
+echo "  3. export KUBECONFIG=\${SHARED_DIR}/kubeconfig"
+echo "  4. oc get nodes"
+echo ""
+echo "To continue tests, create the signal file:"
+echo "  touch /tmp/continue"
+echo "=========================================="
+
+elapsed=0
+while [ ! -f "/tmp/continue" ] && [ $elapsed -lt $DEBUG_WAIT_TIMEOUT ]
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)